docs/tutorials/visibility_simulator.ipynb

```diff
@@ -57,12 +57,22 @@
   },
```

Comment:
<p>In the second paragraph of the MatVis section, is there a floating "The" at the beginning? Also, second paragraph has an instance of "vis_cpu" that should be replaced with "matvis" (beginning of second sentence).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>nice catches, fixed.</p>